### ISSUE TITLE
[Question] Transfer learning rather than training for custom classes ?

### ISSUE THREAD

**Issue by shazz** [fizyr/keras-retinanet#409]
Hi,
Rather than training the full network for custom classes, would it be difficult to add a transfer learning capability on keras-retinanet, to "only" retrain the classifier for example, based on the any pretrained model (COCO, Pascal-VOC...) ?

Thanks !

**Comment by hgaiser** [comment]
Not at all difficult. Just use the --weights argument with the COCO weights
and you're done.

**Comment by shazz** [comment]
Great ! I just saw there were some Closed issues on this topic. Sorry for that.
Thanks for the time spent on this project.... really useful.

**Comment by shazz** [comment]
Oh... --weights only works for COCO, how to use the pretrained COCO model with custom classes ? (csv)

**Comment by hgaiser** [comment]
--weights loads the weights from a file, it doesn't matter if it's from
COCO or another dataset.

**Comment by hgaiser** [comment]
It's best to keep training until the loss no longer decreases or the mAP no longer increases.

The other best thing to do would be to buy a GPU :p

**Comment by shazz** [comment]
Yeah, I'm currently setting up a monster on the cloud with 4 Tesla V100 GPUs.... should be quicker :) But before I wanted to be sure the training parameters were the good ones.

Thanks !

**Comment by hgaiser** [comment]
Haha yeah that should be quicker. Regarding the parameters, I would use more steps (5000) and more epochs (50?). Personally I don't use freeze-backbone, but that's your call.
